microcosm_caching/tests/test_build_info.py

```diff
@@ -0,0 +1,23 @@
+"""
+Unit tests for build_info
+
+"""
+from hamcrest import assert_that, is_
+from microcosm.api import create_object_graph, load_from_dict
+
+
+class TestBuildInfo:
+
+    def test_cached(self):
```

Comment:
Could you add a test checking that caching still works if there's no build info in the config?

Reply:
Sure.